prisma/schema.prisma

```diff
@@ -108,6 +113,7 @@ enum ChallengePurpose {
   SHARE_RECOVERY
   SHARE_ROTATION
   ACCOUNT_RECOVERY
+  AUTHENTICATION
```

Comment:
@matteyu As you now have a specific table for Passkey challenges, is this `AUTHENTICATION` value used?